tools/ARIAtools/ARIAProduct.py

```diff
@@ -103,7 +103,7 @@ def __init__(self, filearg, bbox=None, workdir='./', verbose=False):
                 # Save polygon in shapefile
                 save_shapefile(os.path.join(workdir,'user_bbox.json'), self.bbox, 'GeoJSON')
                 self.bbox_file=os.path.join(workdir,'user_bbox.json')
-                print("Shapefile %s created for input user bounds."%os.path.join(workdir,'user_bbox.json'))
+                log.info ("Shapefile %s created for input user bounds."%os.path.join(workdir,'user_bbox.json'))
```

Comment:
Let the logger do the string formatting.

```suggestion
                log.info ("Shapefile %s created for input user bounds.", os.path.join(workdir,'user_bbox.json'))
```